### ISSUE TITLE
Add the chrono legionairre equipment back into the server

### ISSUE THREAD

**Issue by carlopault** [HippieStation/HippieStationdeprecated2020#12197]
The chrono legionairre is one of, if not, the rarest antag for someone to encounter. I enjoyed playing as the chrono legionairre one time before and would like to play as it again. Memerino informed me that the chrono legionairre's costume has been removed.

**Comment by DaydreamPuppeh** [comment]
Even if it was in the game, they were always admin only. We also do not allow people to choose what their token goes for and we do not spawn them because they are overpowered and unfun to play against.

**Comment by waterwater1212** [comment]
they're supposed to be overpowered.... as far as I know, their purpose was to remove someone for the game (for a reason) in a fun and interesting way and then fuck off. They can't fight crowds, have to stay still to delete someone, can't teleport with anything other than the gear the start out with, and aren't supposed to delete anyone except their target. Don't know how people play it on hippie though... obviously some sort of deleting spree happens and the timeline fractures into a universe where nobody ever existed.

**Comment by Manly-Dwarf** [comment]
Chrono Legionnaire is not overpowered. The time it takes to kill someone is fuckin forever, the gun its self sucks and requires you stand still, and the only advantage it has is its extremely poorly coded and glitchy teleport.
It remains adminbus only and should be removed from the codebase entirely honestly.

**Comment by waterwater1212** [comment]
perhaps i should remove you from the codebase entirely..?

**Comment by Manly-Dwarf** [comment]
please do

**Comment by carlopault** [comment]
> Even if it was in the game, they were always admin only. We also do not allow people to choose what their token goes for and we do not spawn them because they are overpowered and unfun to play against.

Yes, I know how powerful it is, but the chrono legionairre is an antagonist never before seen for most people, which has value in its own right. The chrono legionairre could make for a fun round because it is never seen and if the person playing it only goes after 1 person, and not the entire crew. Of course the chrono legionairre would only make an appearance every forever as to not make everyone despise the antagonist.

**Comment by sagattyy** [comment]
+1 why remove fun

**Comment by Manly-Dwarf** [comment]
The guy playing Chrono Legionnaire would despise it in seconds because of how much of a buggy nightmare it is. It'd be boring as hell as an actual antagonist.
The teleport shit breaks your camera, it glitches out, it has an arbitrary timer that's never told to you, the gun breaks a lot too and sometimes doesn't show the guy you're erasing, the suit gives you slowdown, it's just a giant mess.

**Comment by carlopault** [comment]
> The guy playing Chrono Legionnaire would despise it in seconds because of how much of a buggy nightmare it is. It'd be boring as hell as an actual antagonist.
> The teleport shit breaks your camera, it glitches out, it has an arbitrary timer that's never told to you, the gun breaks a lot too and sometimes doesn't show the guy you're erasing, the suit gives you slowdown, it's just a giant mess.

It wasn't this buggy when I played as it months ago. Have BYOND updates and Hippie Updates broken it?

**Comment by Manly-Dwarf** [comment]
It's just been a disaster for a long time. I'm not against the concept being expanded and turned into something, but in its current state it's nothing but admin fluff for a good reason.

**Comment by ghost** [comment]
make it something like a medigun that adds a non-depleting chemical that gradually slows you down and eventually dematerializes you,  then you become a noclip ghost that can talk to people in garbled speech.  and technological ethereal jump/timed mark and recall kinda thing

**Comment by carlopault** [comment]
> make it something like a medigun that adds a non-depleting chemical that gradually slows you down and eventually dematerializes you, then you become a noclip ghost that can talk to people in garbled speech. and technological ethereal jump/timed mark and recall kinda thing

But the chrono legionairre relates to time, this isn't time related at all.

**Comment by ghost** [comment]
> > make it something like a medigun that adds a non-depleting chemical that gradually slows you down and eventually dematerializes you, then you become a noclip ghost that can talk to people in garbled speech. and technological ethereal jump/timed mark and recall kinda thing
> 
> But the chrono legionairre relates to time, this isn't time related at all.

just talking mechanically, one can always fluff it in a timey wimey way.  quantum disentanglement particles that gradually cut people away from the time stream,  scaling with amount in blood.
